### PR TITLE
Fix ansible remediation of accounts_max_concurrent_login_sessions.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:
- Fix ansible remediation of `accounts_max_concurrent_login_sessions`. Copy behavior of bash remediation, first try to find configuration files from `/etc/security/limits.d/` with `maxlogins`, if it can't find then it remediates `/etc/security/limits.conf`
- Add test scenarios.

#### Rationale:

~~- I couldn't find any other way of searching for strings in ansible other than using `shell + grep` for the specific purpose of this ansible remediation. If you are aware of something that would get rid of the shell command, please advise.~~
- Reference: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_7_security_technical_implementation_guide/V-72217?version=v2r7

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule accounts_max_concurrent_login_sessions:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product ol8
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-ol8-ds.xml accounts_max_concurrent_login_sessions

File: linux_os/guide/system/accounts/accounts-session/accounts_max_concurrent_login_sessions/ansible/shared.yml [hunk 1]
```diff
@@ -3,13 +3,36 @@
...
 # complexity = low
 # disruption = low
-- (xccdf-var var_accounts_max_concurrent_login_sessions)
+
+{{{ ansible_instantiate_variables("var_accounts_max_concurrent_login_sessions") }}}
+
+- name: Find /etc/security/limits.d files contains maxlogins configuration
+  shell: "grep -q '^[^#]*\\<maxlogins\\>' /etc/security/limits.d/*.conf"
+  register: maxlogins
+  failed_when: False
```
Comment (yuumasato) [inline linux_os/guide/system/accounts/accounts-session/accounts_max_concurrent_login_sessions/ansible/shared.yml:12]: Maybe a `find` task with `contains` and `patterns` would work here?
This macro could be inspiring: https://github.com/ComplianceAsCode/content/blob/master/shared/macros-ansible.jinja#L359

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-session/accounts_max_concurrent_login_sessions/ansible/shared.yml:12]: OMG, I overlooked `contains` option in `find` module. At first, I thought `patterns` was what I wanted, but of course it didn't work. Then I gave up and implemented with `shell`. I'll try `find` again and see what happens. Thank you.

**Comment by mildas** [comment]
**Changes identified:**
Rule accounts_max_concurrent_login_sessions:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product ol8
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-ol8-ds.xml accounts_max_concurrent_login_sessions

**Comment by ggbecker** [comment]
@yuumasato I have updated the remediation with `find` ansible module and it's way better now.

**Comment by redhatrises** [comment]
Nice!

**Pull request merged**